src/backend/utils/indexer.js

```diff
@@ -82,6 +82,7 @@ const search = async (textToSearch, filter = 'post') => {
         fields: createFieldsFromFilter(filter),
       },
     },
+    sort: [{ published: { order: 'desc' } }],
```

Comment:
Should we add a  numeric field type for `published` here to indicate that it's a date? https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_sorting_numeric_fields

I'm not sure if that does anything for us or if leaving it off is OK.

Reply:
Hmmm I might be wrong, but I think that's only necessary for casting to a different type? The code seems to be working without it

Reply:
OK, let's not over-complicate this then.